Utilities/Thread.cpp

```diff
@@ -1134,7 +1134,7 @@ bool handle_access_violation(u32 addr, bool is_writing, x64_context* context)
 		case X64OP_STOS:
 		default:
 		{
-			LOG_ERROR(MEMORY, "Invalid or unsupported operation (op=%d, reg=%d, d_size=%lld, i_size=%lld)", op, reg, d_size, i_size);
+			LOG_ERROR(MEMORY, "Invalid or unsupported operation (op=%d, reg=%d, d_size=%lld, i_size=%lld)", (u32)op, (u32)reg, d_size, i_size);
```

Comment:
If newer version would require casts anyway we might as well switch to [fmtlib](http://fmtlib.net/latest/index.html) or fmtlib-like syntax. ZTHere is already one of the fastest logging libraries (with MT support) based on it out there: https://github.com/gabime/spdlog